src/js/plugin-bridge/PluginSDK.js

```diff
@@ -264,7 +267,7 @@ const getSDK = function (pluginID, config) {
     addStoreConfig,
     dispatch: createDispatcher(pluginID),
     Store: StoreAPI,
-    Hooks,
+    Hooks: hooks,
```

Comment:
We could go through everything and rename this to lowercase, but decided that wasn't worth the effort